src/net/driver/mod.rs

```diff
@@ -82,7 +87,7 @@ impl Reactor {
     }
 
     /// Deregisters an I/O event source associated with an entry.
-    fn deregister(&self, source: &dyn Evented, entry: &Entry) -> io::Result<()> {
+    pub fn deregister(&self, source: &dyn Evented, entry: &Entry) -> io::Result<()> {
```

Comment:
`Evented` shouldn't be part of our pub interface.

Reply:
This module is `pub(crate)`.